client/src/App.js

```diff
@@ -1,13 +1,78 @@
+import React, { useState, useEffect } from "react";
 import "./App.css";
+import VideoList from "./VideoList";
+import VideoForm from "./VideoForm";
+
+const API_URL = "https://full-stack-project-bulj.onrender.com";
+
+const App = () => {
+  const [videos, setVideos] = useState([]);
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("Fetched Data:", data);
```

Comment:
usually console log are nice to know what's data is used. tho it's not best to keep later on.